WORKSPACE

```diff
@@ -466,3 +466,12 @@ maven_jar(
   artifact = "com.yahoo.pulsar:pulsar-client:jar:shaded:1.18"
 )
 # end Pulsar Client
+
+# for pex rules
+git_repository(
+    name = "io_bazel_rules_pex",
+    remote = "https://github.com/streamlio/bazel_rules_pex.git",
```

Comment:
Before creating a twitter fork we should look into whether we can proxy this in twitter's env. The fork is not a good solution since it will not automatically get new tags on it, so the build will fail when version updates happen without manual intervention.